.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "software/linux"]
 	path = software/linux
 	url = https://github.com/firesim/linux.git
+[submodule "software/esp-isa-sim"]
```

Comment:
Is it necessary to submodule esp-isa-sim here?  The  version of `spike` built with esp-tools should have the sha3 extension.